crates/bevy_ecs/src/world/mod.rs

```diff
@@ -320,6 +320,15 @@ impl World {
         Some(EntityRef::new(self, entity, location))
     }
 
+    /// Returns an [`Entity`](crate::entity::Entity) iterator of current entities.
+    /// This is useful in contexts where you only have read-only access to the [World].
```

Comment:
```suggestion
    ///
    /// This is useful in contexts where you only have read-only access to the [`World`].
```

Semantic white space. This affects the tooltips on IDEs; we deliberately separate out the short description on the first line from helpful details.

You also missed the backticks :)

Reply:
Ooph, sorry about that; good to know for the future.